.evg.yml

```diff
@@ -46,6 +48,18 @@ tasks:
     commands:
       - func: "build jdbc driver"
 
+  - name: "unittest"
+    depends_on:
+        "build"
```

Comment:
I would suggest that we remove the dependency on the `build` task here, for the integration tests, and for the tdvt tests.
These tasks don't actually depend on any artifacts generated by `build`, so I think all this really does is increase the makespan of patches

Reply:
Removed the dependencies on `build` for unit, integration, and tdvt tests.